man/fi_shm.7.md

```diff
@@ -108,8 +108,8 @@ EPs must be bound to both RX and TX CQs.
 No support for counters.
 
 # RUNTIME PARAMETERS
-
-No runtime parameters are currently defined.
+*FI_SHM_USE_CMA*
+: Controls enabling and disabling of CMA in shm environment. Enabled by default.
```

Comment:
An end-user likely won't have any idea what CMA actually is or why they would need to change this value.  The value isn't enabled by default, but set dynamically.  Given the use, I would rename this to DISABLE_CMA, and define that CMA (Cross Memory Attach - provide this for searching) is a Linux feature for copying data directly between two process (no intermediate buffering needed), but requires that the processes have access to the full address space of the peer.  yadda yadda yadda